includes/ApplHandlingByStr.php

```diff
@@ -93,6 +172,22 @@ function str_isAccepted ( $stat_str, $user )
 }
 
 
+function str_isForwarded( $stat_str, $user )
```

Comment:
__[code_sniffer]__
* Expected 0 spaces between opening bracket and argument "$stat_str"; 1 found
* Expected 0 spaces between argument "$user" and closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->